assets/js/common/OperationModals/OperationModal.jsx

```diff
@@ -17,6 +17,8 @@ function OperationModal({
   onCancel = noop,
   children,
 }) {
+  const warningMessage = `Trento & SUSE cannot be held liable for damages if system is unable to
```

Comment:
You could directly put the text as children. No real need for having a constant. Less to scroll hehe